Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -856,6 +876,18 @@ protected class ClassWriter : IClassWriter {
           preString = $"{DafnyRuntimeModule}.BigOrdinal.is_nat(";
           postString = ")";
           break;
+        case SpecialField.ID.ArrayLength:
+        case SpecialField.ID.ArrayLengthInt:
+          preString = "len(";
+          if (idParam == null) {
+            postString = ")";
+          } else {
+            if ((int)idParam == 0) {
+              postString = ")";
+            }
+            postString = String.Concat(Enumerable.Repeat("[0]", (int)idParam)) + ")";
+          }
```

Comment:
This isn't needed to pass the tests.

Reply:
It is used by the file `7_Arrays.dfy` test file.